manifests/install.pp

```diff
@@ -2,48 +2,35 @@
 #
 # Install the required packages for SSSD
 #
-# == Parameters:
-#
-# [*install_user_tools*]
-# Type: Boolean
-# Default: true
-#   If true, install the 'sssd-tools' for administrative changes to the SSSD
-#   databases.
-#
 # == Authors
 #
 # * Trevor Vaughan <mailto:tvaughan@onyxpoint.com>
 #
-class sssd::install (
-  Boolean  $install_user_tools = true
-) {
+class sssd::install {
   contain '::sssd::install::client'
 
-
-  if ( $::operatingsystem in ['RedHat','CentOS'] ) and ( $::operatingsystemmajrelease > '6' ) {
```

Comment:
I don't see this logic reflected in the YAML files.

The way it is now, we're going to have to create a new YAML file for each release that comes out. We may want to have 6 and then 'default' or something.

Also, this verges on the realm of 'magic' when debugging time comes along for users.